app/layout.tsx

```diff
@@ -26,26 +26,24 @@ export default function RootLayout({ children }: { children: React.ReactNode })
     <html lang="en" suppressHydrationWarning>
       <body className={inter.className}>
         <SettingsProvider>
-          <AppStateProvider>
```

Comment:
Keep the `AppStateProvider` I've added that to share application state. Similar to SettingsProvider, but it's not stored in LocalStorage